imap/ical_support.c

```diff
@@ -2368,7 +2368,7 @@ EXPORTED void icaltimezone_truncate_vtimezone_advanced(icalcomponent *vtz,
     icaltimetype start = *startp, end = *endp;
     icalcomponent *comp, *nextc, *tomb_std = NULL, *tomb_day = NULL;
     icalproperty *prop, *proleptic_prop = NULL;
-    struct observance tombstone;
+    static struct observance tombstone;
```

Comment:
Could we instead require callers to pass a `struct observance*` instead?

Reply:
Fixed